src/components/DataListContainer.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <v-container fluid>
```

Comment:
adding `fluid` did not make any change in the UI. Check once and remove it if it is not making any difference.